test/editor/Core/UtilityTests.re

```diff
@@ -0,0 +1,33 @@
+open TestFramework;
+
+module Utility = Oni_Core__Utility;
+open Utility;
```

Comment:
Perhaps, it's possible to do `open Oni_Core__Utility` directly?

Reply:
Ah, good question... We should perhaps expose `Utility` on the `Oni_Core` module, like we do for the other ones: https://github.com/onivim/oni2/blob/6899c1af8556adc37650405499a588c96c4bcc20/src/editor/Core/Oni_Core.re#L31

This would let us do `module Utility = Oni_Core.Utility` or just `open Oni_Core`.

Reply:
I think it’s already exposed. 🙂

https://github.com/onivim/oni2/blob/6899c1af8556adc37650405499a588c96c4bcc20/src/editor/Core/Oni_Core.re#L21

Reply:
Oh geez totally missed it, was looking at `Uri` and thought they were arranged alphabetically... oops 😄 

In that case, I think at least the `Oni_Core.Utility` is preferred (the double-underscored  `Oni_Core__Utility` is I believe intended to be hidden / private by the `dune` build tool)